frontend/src/pages/instance/AuditLog.vue

```diff
@@ -39,8 +39,9 @@ export default {
         instance () {
             this.$refs.AuditLog?.loadEntries()
         },
-        'team.id' () {
-            this.loadUsers()
+        'team.id': {
+            handler: function (teamId) { if (teamId) this.loadUsers() },
```

Comment:
addressed a bug where team members were not loaded when the page loaded, resulting in an empty users list